### PR TITLE
Better attribution in footer and license

### LINKED ISSUE #183: Clarify license of site and content
This issue was filed because we got a message from a user who said...

>  I liked the new OpenDataKit website and interested using it for CBS. Any restriction on the license?

### PR DESCRIPTION
**Pull request by yanokwa**
Closes #183 

#### What is included in this PR?
* Attribution in footer text
* Additions to LICENSE.md to be extra compliant to various licenses

Confirmed this runs and looks decent locally.

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1 +1,23 @@
-This work is licensed under a [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0/).
\ No newline at end of file
+This site is powered by [Jekyll](https://jekyllrb.com) and [Minimal Mistakes](https://mmistakes.github.io/minimal-mistakes) which are both available under the [MIT License](https://opensource.org/licenses/MIT).
+
+All additional code on this site is licensed under the [Apache 2 License](https://opensource.org/licenses/Apache-2.0) license. All additional content of this site is licensed under a [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0/).
+
+**MIT License**
+
+Copyright 2018 ODK Community
```
Comment (lognaturel) [inline LICENSE.md]: MIT only applies to Jekyll and MM, no?

Comment (yanokwa) [reply LICENSE.md]: You are correct. Will fix!

**Comment by yanokwa** [comment]
@wbrunette We don't put anthing about MIT/Apache in the footer, we just link to the tools we use (which isn't required, but is nice. The long license text is only in the repo.

screen shot 2018-07-24 at 13 40 44

**Comment by wbrunette** [comment]
makes more sense now :)

**Pull request merged**